app/assets/css/pages/_support.styl

```diff
@@ -1,53 +1,2 @@
 .support
-    @extends .split-page
-    #content
-        padding: 0
-        section
-            background-image: image("support/community.png")
-            @media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min--moz-device-pixel-ratio: 2),only screen and (min-device-pixel-ratio: 2),only screen and (min-resolution: 192dpi),only screen and (min-resolution: 2dppx)
-                background-image: image("support/community@2x.png")
-            &:last-child
-                background-image: image("support/pro.png")
-            @media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min--moz-device-pixel-ratio: 2),only screen and (min-device-pixel-ratio: 2),only screen and (min-resolution: 192dpi),only screen and (min-resolution: 2dppx)
-                &:last-child
-                    background-image: image("support/pro@2x.png")
-            p
-                text-align: center
-
-.code
```

Comment:
Just to add another comment that will make sense:

This section was moved to its own file.